blog.py

```diff
@@ -316,13 +320,20 @@ def _load_post(self, post):
             content,
             empty,
             0,
+            name_title_mismatch,
         ])
-    
+
     def _is_ess_empty(self, lines: List[str]) -> int:
         content = ''.join(line.strip() for line in lines)
         is_empty = re.sub(r'\*\*.+\*\*', '', content).replace('**', '') == ''
         return 1 if is_empty else 0
 
+    def _is_name_title_mismatch(self, path, title) -> int:
+        file_name = os.path.basename(path)
```

Comment:
I think there might be some issue there. Dash can exist in title. This might be the reason that you found lots of mismatches. The correct way is to replace each dash in title with a space before comparison.